frontend/rust-lib/flowy-grid/src/event_map.rs

```diff
@@ -111,12 +184,28 @@ pub enum GridEvent {
     #[event(input = "GridCellIdPB", output = "GridCellPB")]
     GetCell = 70,
 
+    /// [UpdateCell] event is used to update the cell content. The passed in data, [CellChangesetPB],
+    /// carries the changes that will be applied to the cell content by calling `update_cell` function.
+    ///
```

Comment:
I'm not clear, here.  What is the update_cell function?  Is that the function that will be ultimately called in Rust?  Because we haven't mentioned those calls in any other part of the documentation for this file.

/// [UpdateCell] event is used to update the content of a cell. The passed in data, [CellChangesetPB],
/// carries the changes that will be applied to the cell content.